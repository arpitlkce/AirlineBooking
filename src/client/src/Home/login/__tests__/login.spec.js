import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import Login from '../login';
const mockStore = configureMockStore();
const store = mockStore({});

describe('Login Component', () => {
    it('should render correctly in  mode', () => {
        let dismissMessage = true;
        let authDetails = [{
            'auth': true,
            'message': 'Success'
        }];
        let location = {state: null};
        const component = renderer.create(<Provider store={store}><Login
            dismissMessage={dismissMessage} authDetails={authDetails} location={location} /></Provider>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
