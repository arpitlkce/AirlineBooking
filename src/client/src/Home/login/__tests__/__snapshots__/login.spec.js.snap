// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component should render correctly in  mode 1`] = `
<div>
  <div>
    <p
      className="lead heading"
    >
      Go Fly AirTicket Booking System
    </p>
    <hr
      className="my-2"
    />
    <div
      className="app container"
    >
      <form
        className="form"
      >
        <div
          className="form-row"
        >
          <div
            className="col"
          >
            <div
              className="row form-group"
            >
              <label
                className="col-sm-4 col-form-label"
                htmlFor="name"
              >
                User Name
              </label>
              <div
                className="col-sm-4"
              >
                <input
                  className="form-control"
                  id="userName"
                  name="userName"
                  onChange={[Function]}
                  type="text"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            <div
              className="row form-group"
            >
              <label
                className="col-sm-4 col-form-label"
                htmlFor="name"
              >
                Password
              </label>
              <div
                className="col-sm-4"
              >
                <input
                  className="form-control"
                  id="password"
                  name="password"
                  onChange={[Function]}
                  type="password"
                />
              </div>
            </div>
          </div>
        </div>
        <button
          aria-label={null}
          className="btn btn-secondary"
          onClick={[Function]}
          type="button"
        >
          LogIn
        </button>
      </form>
    </div>
  </div>
</div>
`;
